src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -181,9 +181,22 @@ export default class ItemDetailPage extends React.PureComponent<Props, State> {
                       </span>
                       <div className="urn-actions">
                         {isThirdParty(item.urn) ? (
-                          <span className="link" onClick={this.handleEditURN}>
-                            {t('item.edit_urn')}
-                          </span>
+                          <Popup
+                            content={t('collection_item.cannot_edit_urn')}
+                            disabled={false}
```

Comment:
The popup should be disabled when the item is not published to prevent it from showing every time, right?
```suggestion
                            disabled={item.isPublished!}
```

Reply:
whooooa, great catch, I forgot to change the `false`, I was testing how it looked and forgot to put the right logic back